config.go

```diff
@@ -13,7 +13,6 @@ type config struct {
 	Namespace    string
 	PollInterval int
 	Client       *redis.Client
```

Comment:
Do we need `config` anymore? Based on these changes, the Redis client can be rolled into `Options` as a private field.

Reply:
Maybe combine `Options` and `config` but keep `Config` as the name?

Reply:
The original intent behind having two separate types was that we do some calculations on the `Options` before it's ready for use as a `config`, and I didn't want people putting unfinished `Options` into the `Config` variable. That said, since `Config` is dead, and `config` is private, I think this distinction is a little less useful. I'll clean it up.